describe.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  describePrivate,
+  PrivateDescription,
+} from "./describe/describePrivate.js";
+
+export type Description = PrivateDescription;
+
+const describeSCLElements: Record<string, (element: Element) => Description> = {
```

Comment:
The type `Record` of `string` is not entirely accurate here, since we don't have entries for every imaginable `string`. One solution would be to just write `Partial<Record<string...`.

Another solution would be to just leave out the type annotation entirely, then the type would be inferred and always 100% accurate.

A much more involved option that might be useful for the development process (rule out typos in tag names) would be to import the `SCLTag` type from a version of `scl-lib`  or `oscd-scl` and use that as the record type instead, representing our goal.

On an unrelated note, what do you think of calling this thing `sclElementDescriptors`?

Reply:
I would propose to go with a partial record here.

Reply:
`sclElementDescriptors` sounds good.